pkg/helm/frrk8s.go

```diff
@@ -153,6 +155,9 @@ func frrk8sValues(envConfig params.EnvConfig, crdConfig *metallbv1beta1.MetalLB)
 		frrk8sValueMap["restartOnRotatorSecretRefresh"] = nil // the cert rotator isn't started anyways
 	}
 
+	if crdConfig.Spec.FRRK8SConfig != nil {
+		frrk8sValueMap["alwaysBlock"] = strings.Join(crdConfig.Spec.FRRK8SConfig.AlwaysBlock, ",")
```

Comment:
I think we should make this stable (sort the list first)

Reply:
done